SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -1067,7 +1071,6 @@ protected void _queueConversationEventsCommitedByUser(SwrveConversation conversa
                 }
                 queueEvent("event", parameters, payload, false);
             }
-            saveCampaignSettings();
```

Comment:
No, that one actually changes the impressions in the campaign. These others were not changing the state, they were saving it because it copied that method but they don't need to at all